src/main/java/frc/robot/AutoCenter.java

```diff
@@ -0,0 +1,54 @@
+package frc.robot;
+
+import com.ctre.phoenix6.mechanisms.swerve.SwerveRequest;
+
+import edu.wpi.first.wpilibj.Timer;
+import edu.wpi.first.wpilibj2.command.Command;
+
+public class AutoCenter extends Command {
+  
+  boolean end = false;
+  double kP = -0.05;
+  private final Timer timer = new Timer();
```

Comment:
Can a comment be added to exactly what the timer is doing? It looks like either the command finishes moving to the point or 0.5 seconds has elapsed? Seems like a short amount of time. Can this be made a configurable parameter?